mackup/applications/transmission.cfg

```diff
@@ -4,3 +4,8 @@ name = Transmission
 [configuration_files]
 Library/Preferences/org.m0k.transmission.plist
 Library/Application Support/Transmission/blocklists
+.config/transmission/blocklists
+.config/transmission/resume
+.config/transmission/torrents
```

Comment:
Sorry but mackup is not a NAS, it's for storing config, not data.
If you do this, your dropbox will constantly sync.